src/css/base.scss

```diff
@@ -358,88 +361,241 @@ p {
   text-align: center;
   font-size: 125%;
   margin-bottom: 1%;
-  // font-family: $cursive-handle-font-family;
 }
 
 .planning-cost {
   text-align: center;
-  font-size: 135%;
+  font-size: 105%;
```

Comment:
These are huge percentages! You could try using 'ems' instead of '%' !

Reply:
Thank you, I will work on it !